payjoin/src/version.rs

```diff
@@ -0,0 +1,39 @@
+use core::fmt;
```

Comment:
I think this is a dup of https://github.com/payjoin/rust-payjoin/blob/master/payjoin/src/core/version.rs.
Probably just an artifact of an old rebase. Good to remove